packages/code-du-travail-frontend/src/dsfr/layout/Header.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
```

Comment:
J'ai tenté de retirer le `use client` sur le header mais ce n'est pas possible car on a besoin de changer le header quand on change de page. 
Voici la doc qui explique pourquoi ce n'est pas possible : https://nextjs.org/docs/app/api-reference/file-conventions/layout#layouts-do-not-receive-searchparams